var/spack/repos/builtin/packages/bridger/package.py

```diff
@@ -17,10 +17,9 @@ class Bridger(MakefilePackage, SourceforgePackage):
     sourceforge_mirror_path = "rnaseqassembly/Bridger_r2014-12-01.tar.gz"
 
     version('2014-12-01', sha256='8fbec8603ea8ad2162cbd0c658e4e0a4af6453bdb53310b4b7e0d112e40b5737')
-
-    # TODO: replace this with an explicit list of components of Boost,
-    # for instance depends_on('boost +filesystem')
-    # See https://github.com/spack/spack/pull/22303 for reference
+    #Maintainers: It looks like the following is the dependency here but needs to be verified by maintainers
```

Comment:
Why is this one different? Is it a partial failure?

Reply:
Yes, I think it was one of the initial ones that I encountered failure with gcc-8.3.1 but it works on gcc-4.9.3.